14-microbiome-diversity.Rmd

```diff
@@ -51,7 +51,14 @@ by species abundances.
 **Dominance** indices are in general negatively correlated with
   diversity, and sometimes used in ecological literature. High
   dominance is obtained when one or few species have a high share of
-  the total species abundance in the community.
+  the total species abundance in the community.  
+  
+**Phylogenetic diversity** was first proposed by [@Faith1992], unlike the 
+  diversity measures mentioned above, Phylogenetic diversity (PD) 
+  measure incorporates information from phylogenetic relationships 
+  stored in `phylo` tree between species in a community (sample). The 
+  Faith's PD is calculated as the sum of branch length of all species in 
+  a community (sample).
```

Comment:
Would be worth pushing the introduction of dominance to a letter point? The order seems a bit confusing 

dominance -> diversity -> alpha/beta/...

Wouldn't this bet better?
diversity -> alpha/beta/... -> dominance

Reply:
Alpha and beta diversity are more different concepts. Dominance is in a way a subset of alpha diversity indices (altough it has an inverse sign). In my opinion, alpha diversity/evenness/richness/dominance/rarity belong together and then beta diversity is an entirely different topic of its own.

Reply:
The order could be: 

richness -> evenness -> diversity -> dominance -> rarity

Reply:
Showcasing each type with functions
- Alpha diversity
   - Richness
   - Phylogenetic diversity
   - Evenness
   - Dominance
  
 - Estimating Alpha Diversity
      -  `estimateRichness()`
      -  `estimateDiversity()` # non-phylo
      -  `picante::pd()`           # phylo
      -   `estimateEvenness()`
      -  `estimateDominance()`  Note: can include coverage index in this section
  
 Thoughts?

Reply:
We could add also "rarity" in the end of this list as it may be added, see [mia#59](https://github.com/FelixErnst/mia/issues/59).

Otherwise ok.

Reply:
`estimateDiversity` is problematic here since it does not specify whether it is alpha or beta diversity. We could use `estimateAlphaDiversity` instead (already implemented)? Although it does match as nicely with other estimate* method names.